securedrop/loaddata.py

```diff
@@ -116,7 +116,7 @@ def set_source_count(s: str) -> int:
 
 
 def add_journalist(
-    username: str = "",
+    username: str,
```

Comment:
`username` is now a required argument to simplify the logic here (and improve typing).